styles.css

```diff
@@ -0,0 +1,193 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body,
+html {
+  font-family: 'Poppins';
+  margin: 0;
+  padding: 0;
+}
+
+/* Styles for Header Section - Navigation */
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    border-bottom: 1px solid lightgray;
+}
+
+nav {
+    width: 600px;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+nav a {
+    text-decoration: none;
+    color: black;
+}
+
+.blackHeart {
+    width: 20px;
+    margin-right: 1rem;
+}
+
+/* Header section - search bar */
+
+.search {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin: 10px
+}
+
+.search a {
+    text-decoration: none;
+    background-color: rgba(0,0,0,.05);
+    color: black;
+    padding: 2px 20px 0px 20px;
+    margin: 0px 10px 0px 0px;
+}
+
+form #search-bar {
+    background-color: rgba(0,0,0,.05);
+    font-size: 1em;
+    width: 980px;
+    height: 30px;
+    padding: 0 8px;
+    box-sizing: border-box;
+    border: none;
+}
+
+#search-button {
+    background-color: #fb3e44;
+    font-size: 1em;
+    color: white;
+    padding: 5px 20px;
+    border: none;
+    margin: 0px 48px 0px 0px;
+    height: 30px;
+}
+
+.logo {
+    width: 50px;
+    height: 20px;
+    display: flex;
+    margin: 0.5rem 1rem 0 0 ;
+}
+
+/* Styles for first section*/
+
+h1 {
+    font-size: 3em;
+    font-weight: bold;
+}
+
+.introduction {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgba(232,217,217,.3);
+    padding: 50px 0 100px;
+}
+
+.npm-description {
+    width: 40%;
+    text-align: center;
+    line-height: 2em;
+    padding-bottom: 50px;
+}
+
+.button-plans {
+    background-color: #FB3B49;
```

Comment:
Try to use a consistent way of declaring colours (Hexadecimal/RGBA/"human language").